specviz/widgets/plotting.py

```diff
@@ -155,8 +135,8 @@ class PlotWidget(pg.PlotWidget):
         This overrides the individual plot data item visibility on
         initialization of the plot widget.
 
-    Attributes
-    ----------
+    Signals
```

Comment:
This should be `Attributes` not `Signals` since the latter is not recognized in the doc building.

Reply:
Done.